crates/environ/src/vmoffsets.rs

```diff
@@ -11,6 +26,11 @@ use cranelift_wasm::{
 use more_asserts::assert_lt;
 use std::convert::TryFrom;
 
+/// Sentinel value indicating that wasm has been interrupted.
+// Note that this has a bit of an odd definition. See the `insert_stack_check`
+// function in `cranelift/codegen/src/isa/x86/abi.rs` for more information
+pub const INTERRUPTED: i32 = (u32::max_value() - 32 * 1024) as i32;
```

Comment:
It's surprising that this is being represented as a signed `i32`, since the value becomes negative, and elsewhere it's casted to a `usize` and treated as unsigned.

Reply:
Er I'm basically trying to be too clever by half here, I've updated this to be `usize`.